### DISCUSSION TITLE
部署到pip install -r requirements.txt发生错误

### DISCUSSION THREAD

**Discussion by tdwaofei** [pengxiao-song/LaWGPT#117] [Q&A（问答）]
pip install -r requirements.txt时报错误，说里面的一个地址：“git+https://github.com/huggingface/peft.git@e536616888d51b453ed354a6f1e243fecb02ea08”不存在

<img width="897" alt="企业微信截图_20230619170708" src="https://github.com/pengxiao-song/LaWGPT/assets/49780642/6e3ce8ab-9a49-475a-b574-4d50f79b3396">

**Comment by tdwaofei** [comment]
在issue里找到了回答：“国内网络环境问题。你可以把requirements.txt里面的github.com替换成kgithub.com（这是一个github镜像网站）
<img width="480" alt="企业微信截图_20230620091117" src="https://github.com/pengxiao-song/LaWGPT/assets/49780642/4b4780ef-aeb7-4d01-bfd9-c254004fc959">
”

**Comment by cqwyj2000** [comment]
还有一种方法，直接到github网站，下载对应版本的zip代码，然后进入代码目录，在你需要安装这些包的conda环境下 运行pip install .
